index.js

```diff
@@ -2,7 +2,10 @@ module.exports = stringify
 
 function stringify (obj) {
   decirc(obj, '', [], null)
-  return JSON.stringify(obj)
+  var hasOpts = arguments.length > 1
+  return (hasOpts)
+    ? JSON.stringify.apply(JSON, arguments)
```

Comment:
`apply` may have a slight perf hit - if that's not super important happy to use the apply always.